### PR TITLE
Gives the hotel staff some fine silk threads, and adds a new hat

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request

***Adds style to the fine hotel, you can tell by a mile!***
Also adds a new hat to wardropes 

## Why It's Good For The Game

Hotel staff looking like fancy assistants makes it hard to tell them apart other then their mindshields. All should work on corgis

## Changelog
:cl:
add: new clothing for the hotel staff and a hat
/:cl:

### CODE REVIEW

File: code/datums/dog_fashion.dm [hunk 1]
```diff
@@ -174,3 +174,13 @@
...
+
+/datum/dog_fashion/head/colour
+	name = "Styled REAL_NAME"
+	desc = "From the tips of their paws to the top of their head, they look like a made bed."
+	emote_see = list("tries to tap dances.","looks sadly at others outfits...","barks at bad fashion!")
+
+/datum/dog_fashion/head/telegram
+	name = "Messager REAL_NAME"
+	desc = "Dont shoot the messager..."
+	emote_see = list("licks an evolope.","looks ready to set off to send a letter...","works on barking!")
```
Comment (Ghommie) [inline code/datums/dog_fashion.dm]:
```suggestion
	emote_see = list("licks an envelope.","looks ready to set off to send a letter...","works on barking!")
```

File: code/datums/dog_fashion.dm [hunk 2]
```diff
@@ -174,3 +174,13 @@
...
 	name = "Trooper REAL_NAME"
 	desc = "That's not red paint. That's real corgi blood."
+
+/datum/dog_fashion/head/colour
+	name = "Styled REAL_NAME"
+	desc = "From the tips of their paws to the top of their head, they look like a made bed."
+	emote_see = list("tries to tap dances.","looks sadly at others outfits...","barks at bad fashion!")
+
+/datum/dog_fashion/head/telegram
+	name = "Messager REAL_NAME"
```
Comment (Ghommie) [inline code/datums/dog_fashion.dm]:
```suggestion
	name = "Messanger REAL_NAME"
```

File: code/datums/dog_fashion.dm [hunk 3]
```diff
@@ -174,3 +174,13 @@
...
+	desc = "Dont shoot the messager..."
```
Comment (Ghommie) [inline code/datums/dog_fashion.dm]:
```suggestion
	desc = "Dont shoot the messanger..."
```

File: code/datums/dog_fashion.dm [hunk 4]
```diff
@@ -174,3 +174,13 @@
 /datum/dog_fashion/back/deathsquad
 	name = "Trooper REAL_NAME"
 	desc = "That's not red paint. That's real corgi blood."
+
+/datum/dog_fashion/head/colour
+	name = "Styled REAL_NAME"
```
Comment (Ghommie) [inline code/datums/dog_fashion.dm]:
```suggestion
	name = "Stylish REAL_NAME"
```

File: code/datums/dog_fashion.dm [hunk 5]
```diff
@@ -174,3 +174,13 @@
...
+	desc = "From the tips of their paws to the top of their head, they look like a made bed."
```
Comment (Ghommie) [inline code/datums/dog_fashion.dm:180]: Never heard of this simile or metaphor.

File: code/modules/clothing/under/miscellaneous.dm [hunk 6]
```diff
@@ -693,13 +766,15 @@
...
 	can_adjust = FALSE
+
 /obj/item/clothing/under/gondola
 	name = "gondola hide suit"
 	desc = "Now you're cooking."
 	icon_state = "gondola"
 	item_state = "lb_suit"
 	item_color = "gondola"
 	can_adjust = FALSE
+
```
Comment (Ghommie) [inline code/modules/clothing/under/miscellaneous.dm:778]: Lot of whitespaces, sure, but it's very irritating how the paths lacked spacing from one other, so ok.

**Review by Ghommie — COMMENTED** [review]
Usual painful gremmar from trilby. :man_shrugging:

File: code/datums/dog_fashion.dm (hunk 5 above)
Comment (Trilbyspaceclone) [reply code/datums/dog_fashion.dm:180]: A made bed looks nice

File: code/modules/clothing/under/miscellaneous.dm (hunk 6 above)
Comment (Trilbyspaceclone) [reply code/modules/clothing/under/miscellaneous.dm:778]: They were removed somewhat and lots of random white space were old flags were got removed.
But I frankly dont know why this file and only this one had no white spaces were new items were

File: code/datums/dog_fashion.dm (hunk 5 above)
Comment (Putnam3145) [reply code/datums/dog_fashion.dm:180]: it's understandable i think it's fine

Comment (Thalpy) [reply code/datums/dog_fashion.dm:180]: Trilby makes the best Limericks

**Comment by Thalpy** [comment]
Is it not messenger, not messanger ghommie..?

**Comment by Ghommie** [comment]
> 
> 
> Is it not messenger, not messanger ghommie..?

Fuck, you are right.

**Review by Linzolle — APPROVED** [review]

**Pull request merged**